001-intro.md

```diff
@@ -100,96 +100,358 @@ to real life cases involving actual DMs.**
-
-**VO-enabled plotting and fitting applications.** An application whose
+Note that the same file may contain multiple tables and in any case
+STC model instances defined in multiple coordinate systems or frames.
+
+### VO-enabled plotting and fitting applications ###
+An application whose
 main requirement is to display, plot, and/or fit data cannot be required
-to be aware of *all* data models. However, if these data models share
+to be aware of *all* astronomical data models. However, if these data models shared
 some common representation of quantities, their errors, and their units,
-the application can discover these pieces of information and structure a
-plot, or perform a fit, with minimal user input: each point will be
+the application may discover these pieces of information and structure a
+plot, or perform a fit, with minimal user input: each point may be
 associated with an error bar, upper/lower limits, and other metadata.
-The application remains mostly Data Model-agnostic: it wouldn’t need to
+The application remains mostly model-agnostic: it is not required to
 *understand* high-level concepts like Spectrum, or Photometry.
 
-**Validators.** The existence of an explicit Data Model representation
+Also, knowledge about the basic building-block types like quantities
+and coordinates, may be hard-coded during development.
+
+### Data discovery portals ###
+Data discovery portals allow users to query VO services, display metadata,
+filter responses, and fetch datasets.
+
+While these applications may not be particularly interested in specific data models,
+standardized annotation mechanisms may allow their developers to dynamically capture
+the structure of the metadata, and provide better exploratory tools rather than flat
+tables.
+
+Consider for example filtering tables according to an arbitrary physical quantity, say
+for instance the spectral coverage of a spectrum, the filter with which an image was taken, or
+the luminosity of a source in a certain band. The portal may provide a friendly interface that allows
+users to select the physical quantities using standardized representations. It may do
+so dynamically for all the pieces of metadata present in the dataset, rather
+than limiting this functionality to a set of hard-coded metadata properties.
+
+Also, the application may group data and metadata in a tree of concepts the user is familiar with,
+rather than flattening the instrinsic structure of the data.
+
+By allowing such applications to faithfully represent data model instances the way they
+were curated and annotated by data providers, and according to the scientific domain
+with which users are familiar, this specification may enable users to easily make sense of
+the file contents, even if the application lacks any knowledge of high-level data models.
+
+Parsing VO-DML description files may be useful to provide the user with even more, and more
+accurate, information.
+
+### Color-color diagrams ###

```

Comment:
@lmichel
Your comment:
> There are lots of similarities between 2.2.3 et 2.2.4. They could be merged

If you are referring to this use case and the one before (Data discovery portals and color-color diagrams) I am not convinced they are really similar, if not in a very abstract sense, but the goal of these sections is to provide concrete examples of the abstract cases. I wouldn't know how to merge these two use cases.